.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <annotationProcessing>
+      <profile name="Maven default annotation processors profile" enabled="true">
+        <sourceOutputDir name="target/generated-sources/annotations" />
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <module name="figures" />
+      </profile>
+    </annotationProcessing>
+  </component>
+</project>
```

Comment:
this file shouldn't be here 
you probably first have added it to staging area of git or even committed this file, and then added it to .gitignore
In this case try to follow this instruction to remove it from PR 
https://stackoverflow.com/a/14797830